Added DocBook snippets as a separate extension (#9)
openSUSE/suse-vscode-doc: extensions/docbook-snippets/vsc-extension-quickstart.md

# Welcome to your VS Code Extension

## What's in the folder

> * This folder contains all of the files necessary for your extension.

Reviewer: Who is the target user? The developer writing this extension or the user using it?

Maybe add your target group.

> * This folder contains all of the files necessary for your extension.
> * `package.json` - this is the manifest file that defines the location of the snippet file and specifies the language of the snippets.
> * `snippets/snippets.code-snippets` - the file containing all snippets.
> 

Reviewer 2: ```suggestion
The folder contains the required components of the extension.
* `package.json` - the manifest file that defines the location of the snippet file and specifies the language of the snippets.
* `snippets/snippets.code-snippets` - the file containing snippets.

```

> ## Get up and running straight away

Reviewer 2: What does _Get up and running_ mean? Build an extension? Install it? Run it? Something else?

Also, the hed sounds a bit colloquial to me

> * Press `F5` to open a new window with your extension loaded.

Reviewer 2: Press where? In VS Code? The extension loaded? When and how did that happen?

> * Create a new file with a file name suffix matching your language.

Reviewer 2: An example would make this otherwise confusing sentence more clear.

> * Create a new file with a file name suffix matching your language.

Reviewer 2: An example would make this otherwise confusing sentence more clear.

> * Verify that your snippets are proposed on IntelliSense.

Reviewer 2: How do I verify that?

## Make changes

* You can relaunch the extension from the debug toolbar after making changes to the files listed above.
* You can also reload (`Ctrl+R` or `Cmd+R` on Mac) the VS Code window with your extension to load your changes.

> ## Install your extension
> 
> * To start using your extension with Visual Studio Code copy it into the `<user home>/.vscode/extensions` folder and restart Code.
> * To share your extension with the world, read on https://code.visualstudio.com/docs about publishing an extension.

Reviewer 2: ```suggestion
## Install the extension

* To use the extension with Visual Studio Code or VSCodium, copy it into the `<user home>/.vscode/extensions` folder and restart the application.

To publish the extension so other can install and use it, refer to the [official documentation](https://code.visualstudio.com/docs)
```

Reviewer: I assume a link would also work. Maybe it would be easier or more efficient to store the repo somewhere and link to it in your user home?

That way the user can update the repo without forgetting to copy it? Or should better kept separately?
